Source/CarthageKit/Project.swift

```diff
@@ -458,6 +459,21 @@ public final class Project {
 				return copyProduct(dSYMURL, resolvedDestinationURL)
 			}
 	}
+	
+	/// Copies any *.bcsymbolmap files matching the given framework and contained
+	/// within the given directory URL to the directory that the framework
+	/// resides within.
+	///
+	/// If no bcsymbolmap files are found for the given framework, completes with
+	/// no values.
+	///
+	/// Sends the URLs of the bcsymbolmap files after copying.
+	public func copyBCSymbolMapsToBuildFolderForFramework(frameworkURL: NSURL, fromDirectoryURL directoryURL: NSURL) -> SignalProducer<NSURL, CarthageError> {
+		return BCSymbolMapsForFramework(frameworkURL)
```

Comment:
On a similar note, Carthage already assumes the dSYM will be in the same folder as the framework, and that's not seen as a problem. Sure, if the dSYM is missing, Xcode won't fail to archive, but I think that's an argument in favor of ensuring your bcsymbolmap files are in the same folder as your framework, rather than, well, I'm not sure what the alternative is anyway, unless you want to search the entire archive for the file.